documentation/source/getting_started/features.rst

```diff
@@ -1,8 +1,8 @@
 .. _Getting-Started:
 Features
 ====================
-OpenEBS offers persistent block storage with the following features:
+OpenEBS offers persistent block storage with the following features.
 
-- Containerized block storage using Docker containers. We call them volumes or Virtual Storage Machines, a concept similar to k8s PODs.
-- A highly scalable storage orchestration platform that spins the storage volumes seamlessly and manages them effortlessly
-- A simple, yet high performing distributed block storage designed with best caching via the NVMe optimizations
+* Containerized block storage using Docker containers. They are called volumes or Virtual Storage Machines, similar to k8s PODs.
```

Comment:
The above line can be changed to : containerized storage for containers, that allows devops teams - working with stateful containers to have their own customized storage controllers.

(We need to remove the references to VSM )

Reply:
Done!